packages/mobile/locales/en-US/translation.json

```diff
@@ -1078,6 +1078,10 @@
   "personaAccountEndpointFail": "Identity verification is not currently available. Please try again later",
   "linkBankAccountScreen": {
     "tryAgain": "Try Again",
+    "verifying": {
+      "title": "Verifying your Identity",
+      "description": "Please keep the app open!"      
```

Comment:
why does the app need to stay open?

Reply:
Also curious about this; will something "go wrong" if the user closes out of the app during this?

Reply:
Good question. The app at this point is just waiting for IHL to update the kycStatus in firebase. Maybe not necessary?